app/test_main.py

```diff
@@ -2,0 +1,29 @@
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_access_google_page_with_true_false(
+        mock_valid_url: bool,
+        mock_connection: bool
+) -> None:
+
+    mock_valid_url.return_value = True
+    mock_connection.return_value = False
+
+    assert can_access_google_page("") == "Not accessible"
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_access_google_page_with_false_true(
+        mock_valid_url: bool,
+        mock_connection: bool
```

Comment:
Similar to the previous test case, the mock parameters `mock_valid_url` and `mock_connection` should be of type `mock.Mock`, not `bool`.